### PR TITLE
More product attributes

### PR DESCRIPTION
**Pull request by Sinetheta**

### CODE REVIEW

**Comment by danmitchell-** [comment]
I'd like to see these changes get merged...

File: spec/models/spree/feed_product_spec.rb [hunk 1]
```diff
@@ -43,6 +51,26 @@
...
+
+    context "when the product has one image" do
+      before { Spree::Image.create! viewable: product.master, attachment_file_name: 'hams.png' }
+      it { is_expected.to eq [] }
+    end
+
+    context "when the product has more than one image" do
+      before { Spree::Image.create! viewable: product.master, attachment_file_name: 'hams.png' }
+      before { Spree::Image.create! viewable: product.master, attachment_file_name: '2hams.png' }
+      before { binding.pry }
```
Comment (jarednorman) [inline spec/models/spree/feed_product_spec.rb]: oopsy

**Comment by jarednorman** [comment]
@Sinetheta What's the status of this? Do you intend to get it over the finish line?

**Comment by Sinetheta** [comment]
I can, but I think we might want to evaluate https://github.com/gevann/solidus_product_feed instead.

**Pull request closed without merging**